src/reconcile/rec_write.c

```diff
@@ -2091,6 +2091,47 @@ __rec_delta_pack_key(WT_SESSION_IMPL *session, WT_BTREE *btree, WT_RECONCILE *r,
     return (ret);
 }
 
+/*
+ * __wti_rec_pack_delta_internal --
+ *     Pack a delta for an internal page
```

Comment:
Sorry to nitpick but the comment would make a little more sense if it ended `... for an internal page into a reconciliation structure`.